python_modules/libraries/dagster-airlift/dagster_airlift/core/components/airflow_instance/component.py

```diff
@@ -75,6 +76,27 @@ def resolve_mapped_asset(context: ResolutionContext, model) -> Union[AssetKey, A
 ]
 
 
+@dataclass
+class AirflowFilterParams(Resolvable):
+    dag_id_ilike: Optional[str] = None
```

Comment:
if you make this a BaseModel you can add documentation to these params so it'll show up in our tooling